### PR TITLE
Popup implementation

### PR DESCRIPTION
**Pull request by ghost**
Tried to implement popups to the mobile and desktop version of the portfolio.

### CODE REVIEW

File: projects.js [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    id: '3',
+  },
+  {
+    title: 'Multi-Post Stories',
+    company: 'Tonic',
+    position: 'Back End Dev',
+    date: '2015',
+    desc: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    body: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
```
Comment (ShoiraTa) [inline projects.js:41]: - [x] I have noticed that the container of this text has a fixed height, thus the text is going out of container border, please fix this issue: 
image

File: projects.js [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+					<img
+						src="./images/javascript.png"
+						alt="javascript tag"
+						class="work-tags"
+					/>
+					</div>
+              </div>
+              <div class="modal-links">
+                <a href="./index.html"><button class="btn-input">See Live &nbsp;<img src="./images/see-live.svg" alt="See Live"></button></a>
+                <a href="./index.html"><button class="btn-input">See Source &nbsp;<img src="./images/see-source.svg" alt="See Live"></button></a>
```
Comment (ShoiraTa) [inline projects.js:147]: - [x] In smaller screens the text in buttons is wrapping, please try to match the design with Figma design regardless the screen  size:

First Header  | Second Header
------------- | -------------
image  | image

- [ ] Please delete the underline from the text in links

File: projects.js [hunk 3]
```diff
@@ -0,0 +1,165 @@
+const projects = [
+  {
+    title: 'Multi-Post Stories',
+    company: 'CANOPY',
+    position: 'Back End Dev',
```
Comment (ShoiraTa) [inline projects.js:5]: - [x] Please check the popup modal location on screens, currently, it is located not in the center:

image
As you can see there is more space remaining on the right side, please locate it in the center.

File: styles.css [hunk 4]
```diff
@@ -474,6 +474,103 @@
...
+    margin-right: 50px;
+  }
+
+  .modal-close {
+    position: absolute;
+    right: 100px;
+    top: 25px;
+    width: 80px;
+    height: 80px;
+  }
```
Comment (ShoiraTa) [inline styles.css]: - [x] Please justify this button to the end to match the design with Figma:

Figma   | Yours
------------- | -------------
 image | image

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi team,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

## OPTIONAL:

- [ ]  OPTIONAL I would strongly recommend you to add body { overflow: hidden} when the popup modal is visible, this way users will not be able to scroll the screen when the popup is on.

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: projects.js (hunk 2 above)
Comment (ghost) [reply projects.js:147]: I am having a little bit of trouble implementing this one. Is there not a minimum width we should be ready to make our websites work on? my button follows the Figma design up until 395px, are there devices that use less than that?
Thank you for the feedback, I will keep working on it.

Comment (ghost) [reply projects.js:147]: Ok, I made several changes and it is not breaking on screens as small as 360px wide now, and I am not sure how I could improve it further.
Thank you for the feedback.

**Review by mmsesay — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**